ui/com/user-info.jsx

```diff
@@ -25,8 +25,8 @@ const FLAG_DROPDOWN = [
 class AutoRefreshingComponent extends React.Component {
   constructor(props) {
     super(props)
-    this.state = this.computeState()
-    this.refreshState = props => { this.setState(this.computeState(props)) }
+    this.computeState(null, this.setStateCallback.bind(this))
```

Comment:
@pfraze are you suggesting setting that const inside the class definition like : 

```javascript
class AutoRefreshingComponent extends React.Component {
  const handleCB = (err, state) => {
    if (err) throw err
    this.setState(state)
  }
  constructor(props) {
    super(props)
    this.computeState(null, handleCB)
    this.refreshState = props => { this.computeState(props, handleCB) }
  }
...
```
?

I don't know of a way to do that, and pushing the const outside the class means you don't have access to `this` any more. 
I assume I'm missing some basic knowledge D: